src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -241,3 +243,119 @@ def test_load_user_metadata_bad(inputs, msg):
     actual_args = get_args(cli_inputs)
     with pytest.raises(ValueError, match=msg[0]):
         actual_args = load_user_metadata(actual_args)
+
+
+def _setup_dirs(monkeypatch, user_filesystem):
```

Comment:
I think most of these tests are not needed since they are tested in the `diffpy.utils.tools` functions.

We just have to test what this load_user_data does, so give it an arg without the username and have it return an arg with the username.  I guess we could test when the `arg.username` is `None` and when it has something in it, then just create a config file with something in it.  You don't have to test all the different config possibilities which are tested in utils.

Reply:
Ahh got it. In this case we only need to test args with/without username/email and check that arg.username and email are properly loaded?